src/util/getRoutePath.js

```diff
@@ -1,10 +1,14 @@
-export default function getRoutePath(route, routes) {
+export default function getRoutePath(route, routes, name) {
   const matchIndex = routes.indexOf(route);
   if (matchIndex < 0) {
     throw new Error('`route` not found in `routes`');
   }
   const routesUntil = routes.slice(0, matchIndex + 1);
-  return routesUntil
+  let routePath = routesUntil
     .reduce((acc, { path }) => acc.concat(path), [])
     .join('/');
+  if (name) {
+    routePath = `${routePath}@${name}`;
```

Comment:
I like that, seems to also not be used/recommended in URLs.

> The characters "<" and ">" are unsafe because they are used as the delimiters around URLs in free text